client/wfnews-war/src/main/angular/android/app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Shouldn't this file be removed? We have it in secrets as GOOGLE_SERVICES_JSON_BASE64, if it needs to be updated I think the change should be made in there.